ark/segmentation/marker_quantification.py

```diff
@@ -27,11 +29,20 @@ def compute_marker_counts(input_images, segmentation_labels, nuclear_counts=Fals
             morphology features for regionprops to extract for each cell
         split_large_nuclei (bool):
             controls whether nuclei which have portions outside of the cell will get relabeled
+        extraction (str):
+            extraction function used to compute marker counts. default is 'total_intensity'
+        **kwargs:
+            arbitrary keyword arguments
     Returns:
         xarray.DataArray:
             xarray containing segmented data of cells x markers
     """
 
+    misc_utils.verify_in_list(
+        extraction=[extraction],
+        extraction_options=list(extraction_function.keys())
+    )
+
     if regionprops_features is None:
         regionprops_features = ['label', 'area', 'eccentricity', 'major_axis_length',
                                 'minor_axis_length', 'perimeter', 'centroid']
```

Comment:
Candidate for `ark.settings`?

Reply:
Used to be in `ark.settings` before being merged in.  Removed them since the names won't change and it's doubtful that we'll want to include any other props by default.  Not a hassle to put it back in, but that was last I heard on that front.

Reply:
I think it's fine to leave it defined within this function, since it's only ever going to be used in here, not called anywhere else.